Horan_Colby/Assignments/registration_form/registration.py

```diff
@@ -0,0 +1,72 @@
+from flask import Flask
+from flask import render_template
+from flask import session
+from flask import request
+from flask import redirect
+from flask import flash
+
+import re
+EMAIL_REGEX = re.compile(r'^[a-zA-Z0-9.+_-]+@[a-zA-Z0-9._-]+\.[a-zA-Z]+$')
+
+app = Flask(__name__)
+app.secret_key = 'WHAAAAAAAT'
+
+@app.route('/')
+def index():
+	return render_template('/registration.html')
+
+@app.route('/process', methods= ['post'])
+def process():
+	user = {
+		'email' : request.form['email'],
+		'firstname' : request.form['first'],
+		'lastname' : request.form['last'],
+		'password' : request.form['password'],
+		'confirm' : request.form['confirm']
+	}
+	session['email'] = request.form['email']
+	session['first'] = request.form['first']
+	session['last'] = request.form['last']
+	session['password'] = request.form['password']
+	session['confirm'] = request.form['confirm']
```

Comment:
u know, u could just `session['user'] = user` and be done with it in a single line 😞